test/e2e/serial/workload_placement_test.go

```diff
@@ -443,6 +444,128 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 			}, time.Minute, time.Second*5).Should(BeTrue(), "resources not restored on %q", updatedPod.Spec.NodeName)
 		})
 	})
+	Context("cluster with node/s having two numa zones, and there are enough resources on one node but not in any numa zone when trying to schedule a deployment with burstable pods", func() {
+		var nrtCandidates []nrtv1alpha1.NodeResourceTopology
+		var targetNodeName string
+		var targetNodeNRTInitial *nrtv1alpha1.NodeResourceTopology
+		var deployment *appsv1.Deployment
+
+		BeforeEach(func() {
+			const requiredNUMAZones = 2
+			By(fmt.Sprintf("filtering available nodes with %d NUMA zones", requiredNUMAZones))
+			nrtCandidates = e2enrt.FilterZoneCountEqual(nrts, requiredNUMAZones)
+
+			const neededNodes = 1
+			if len(nrtCandidates) < neededNodes {
+				Skip(fmt.Sprintf("not enough nodes with at least %d NUMA Zones: found %d, needed %d", requiredNUMAZones, len(nrtCandidates), neededNodes))
+			}
+
+			nrtCandidateNames := e2enrt.AccumulateNames(nrtCandidates)
+
+			var ok bool
+			targetNodeName, ok = nrtCandidateNames.PopAny()
+			Expect(ok).To(BeTrue(), "cannot select a node among %#v", nrtCandidateNames.List())
+			By(fmt.Sprintf("selecting node to schedule the pod: %q", targetNodeName))
+
+			var err error
+			targetNodeNRTInitial, err = e2enrt.FindFromList(nrtCandidates, targetNodeName)
+			Expect(err).NotTo(HaveOccurred())
+
+			//get maximum zone CPU and Memory to be sure it wont fit on any zone
+			maxResources := corev1.ResourceList{
+				corev1.ResourceCPU:    maxResourceType(*targetNodeNRTInitial, corev1.ResourceCPU),
+				corev1.ResourceMemory: maxResourceType(*targetNodeNRTInitial, corev1.ResourceMemory),
+			}
+
+			// add a mim ammount of resources just in case all the zones
+			// are equal so the pod wont fit on any of them
+			excessRes := corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("100m"),
+				corev1.ResourceMemory: resource.MustParse("100M"),
+			}
+			reqResources := maxResources.DeepCopy()
+
+			reqCpu := reqResources[corev1.ResourceCPU]
+			reqCpu.Add(excessRes[corev1.ResourceCPU])
```

Comment:
it seems to me we can inline the `resourceMustParse` above and remove `excessRes` entirely